tfs/src/main/resources/hudson/plugins/tfs/TeamPRPushTrigger/TeamPollingAction/index.jelly

```diff
@@ -0,0 +1,45 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2009, Sun Microsystems, Inc., Kohsuke Kawaguchi
```

Comment:
Copyright header should be an MS copyright.

Reply:
Removed the whole file.